tests/baselines/reference/paramTagNestedWithoutTopLevelObject4.errors.txt

```diff
@@ -0,0 +1,12 @@
+tests/cases/conformance/jsdoc/paramTagNestedWithoutTopLevelObject4.js(2,20): error TS8032: Qualified name 'xyz.bar.p' is not allowed without a leading '@param {object} xyz.bar'.

```

Comment:
I like this error because, although it's not correct, (1) pasting in the suggested line leads to another error with the correct fix and (2) I can't think of a good way to word the fact that you actually need *two* more lines.